Makefile

```diff
@@ -1,35 +1,17 @@
 build:
-	cp pkg/META.in pkg/META
-	ocaml pkg/build.ml native=true native-dynlink=true
+	jbuilder build
 
-test: build
-	rm -rf _build/src_test/
-	ocamlbuild -j 0 -use-ocamlfind -classic-display \
-						 src_test/test_ppx_yojson.byte --
+test:
+	jbuilder runtest
 
 doc:
-	ocamlbuild -use-ocamlfind doc/api.docdir/index.html \
-						 -docflags -t -docflag "API reference for ppx_deriving_yojson" \
-						 -docflags '-colorize-code -short-functors -charset utf-8' \
-						 -docflags '-css-style style.css'
-	cp doc/style.css api.docdir/
+	jbuilder build @odc
 
 clean:
-	ocamlbuild -clean
+	jbuilder clean
 
 .PHONY: build test doc clean
 
-gh-pages: doc
-	git clone `git config --get remote.origin.url` .gh-pages --reference .
-	git -C .gh-pages checkout --orphan gh-pages
-	git -C .gh-pages reset
-	git -C .gh-pages clean -dxf
-	cp -t .gh-pages/ api.docdir/*
-	git -C .gh-pages add .
-	git -C .gh-pages commit -m "Update Pages"
-	git -C .gh-pages push origin gh-pages -f
-	rm -rf .gh-pages
```

Comment:
I don't think jbuilder provides this.

Reply:
Yes, this will need a bit of fixing. I need to change this bit to push the docs generated by odoc instead. I'll do it if there's general acceptance of the PR.

Reply:
This can also probably be done using `topkg-jbuilder`.